published-namespaces.txt

```diff
@@ -274,6 +291,7 @@ AndroidX.Work.Impl.Utils.Futures
 AndroidX.Work.Impl.Utils.TaskExecutor
 AndroidX.Work.Impl.Workers
 AndroidX.Work.MultiProcess
+Dev.Chrisbanes.Snapper
```

Comment:
Should this be `Xamarin.Dev.ChrisBanes.Snapper` like below?

Reply:
correct. Forgotten to add rule to `Directory.Build.props`